### ISSUE TITLE
Plugin data saved in multiple places

### ISSUE THREAD

**Issue by berthelemy** [getgrav/grav#3537]
This links to issue https://github.com/getgrav/grav/issues/3432

As well as the multiple security salts, I also have one plugin (custom-banner) which is serving up data from two places, depending on how you access the site.

I have several directories like this:
/grav/user/www.[domain]/config
/grav/user/[domain]/config
/grav/user/mail.[domain]/config

Each config directory contains a security.yaml file, with a salt.

In www.[domain]/config, there is also a plugins directory, containing custom-banner.yaml, which contains data for the banner. Let's call this DATA1.

The same file exists in /grav/user/config/plugins. Let's call this DATA2.

If a user accesses the site via www.[domain] they see the data in DATA1.

If a user accesses via [domain], they see the data in DATA2.

Can I just delete these stray config directories in /grav/user?

**Comment by mahagr** [comment]
Yes, there was a bug related to this and the new behavior is that if there's no config folder in the environment, it will not be created (and used) automatically anymore. 

So just go on and delete the folders which shouldn't have their own config.
